### ISSUE TITLE
NPM "version" export format changed (now includes SHA and date)

### ISSUE THREAD

**Issue by webcarrot** [facebook/react#24215]
React version: 18.0.0

## Steps To Reproduce

1. `import { version } from "react"`
2. `console.log(version)`

Link to code example:

https://codesandbox.io/s/new-platform-dvqpp8?file=/src/App.js

## The current behavior

Version is: `18.0.0-fc46dba67-20220329`

## The expected behavior

Version is: `18.0.0` (or `18.0.1` 😉 )

**Comment by gaearon** [comment]
How do you use it?

**Comment by webcarrot** [comment]
> How do you use it?

As a part of CDN link during server side rendering. 

Something like:

```ts
const ENV = process.env.NODE_ENV === "development" ? "development" : "production.min";const REACT_URL = `https://unpkg.com/react@${React.version}/umd/react.${ENV}.js`;const REACT_DOM_URL = `https://unpkg.com/react-dom@${React.version}/umd/react-dom.${ENV}.js`;
```

**Comment by bvaughn** [comment]
I think Dan was asking how you used the `version` attribute exported from a React renderer. The code above looks more like how you choose which React/DOM version to load.

**Comment by gaearon** [comment]
I think there is usage of `React.version` in it so seems valid.

We talked a bit about this and we're wondering — is there any reason you don't want to use `require('react/package.json').version`? This seems like a more accurate guarantee anyway.

**Comment by bvaughn** [comment]
> I think there is usage of `React.version` in it so seems valid.

But it's used _before_ React is imported. After all, they're using it _to determine which URL to load the UMD package from_ so...that seemed like a mistake?

**Comment by gaearon** [comment]
I'm assuming this is the code that's running on the server deciding which `<script>` tags to include in HTML for the client.

**Comment by bvaughn** [comment]
Ah, I see. Fair enough 👍🏼

**Comment by webcarrot** [comment]
I reported this as a bug because I thought this change was a mistake during package build.

If this is not the case, please just close this issue as invalid.

> We talked a bit about this and we're wondering — is there any reason you don't want to use `require('react/package.json').version`? This seems like a more accurate guarantee anyway.

In an EMS environment it's probably not that easy ... anyway now I just do:
```ts
const REACT_VERSION = React.version.split("-")[0];
//...
```
and I'm happy.

**Comment by webcarrot** [comment]
I'm closing this issue as this is the expected behaviour.

**Comment by gaearon** [comment]
I think we were planning to change this in 18.1.

**Comment by gaearon** [comment]
Fixed in 18.1
